genesyscloud/tfexporter/resource_genesyscloud_tf_export.go

```diff
@@ -114,17 +111,17 @@ func ResourceTfExport() *schema.Resource {
 				Elem:        &schema.Schema{Type: schema.TypeString},
 				ForceNew:    true,
 			},
+			"add_depends_on_capability": {
```

Comment:
Consider renaming to `enable_depends_on_for_flows` or `enable_flow_depends_on` or something